relayer/processor/path_processor_internal.go

```diff
@@ -792,8 +944,8 @@ func queuePendingRecvAndAcks(
 	seqs []uint64,
 	srcCache ChannelPacketMessagesCache,
 	dstCache ChannelPacketMessagesCache,
-	srcMu *sync.Mutex,
-	dstMu *sync.Mutex,
+	srcMu sync.Locker,
+	dstMu sync.Locker,
```

Comment:
out of curiosity what's the reasoning behind this change?

Reply:
it was suggested by the linter to use the interface rather than a pointer to the mutex